scripts/dts/gen_defines.py

```diff
@@ -114,8 +114,9 @@ def parse_args():
 
     parser = argparse.ArgumentParser()
     parser.add_argument("--dts", required=True, help="DTS file")
-    parser.add_argument("--bindings-dir", required=True,
-                        help="directory with bindings in YAML format")
+    parser.add_argument("--bindings-dir", nargs='+', required=True,
+                        help="directory with bindings in YAML format, "
+                        "we allow multiple")
```

Comment:
I think `--bindings-dirs`.  Four of the five scripts that use `nargs='+'` use plural nouns in the name.

Reply:
`--binding-dirs` fine too (just moving the `s`)? The double `s` keeps tripping me up.

Reply:
Not really fine with me, no.  The directory is named `bindings`.  When there was one of them, `--bindings-dir` was right.  When there are multiple it should be `--bindings-dirs`.  Name consistency is important for preservation of sanity.

Reply:
Though this is a bikeshed argument.  I'll live with either (but will curse if my file isn't found because I put it in a directory named `binding` that cmake didn't look at).

Reply:
The script doesn't care what the directory is called at least. `--bindings-dir` just meant "this is a directory with bindings" to me, not "this is the directory with the naming bindings/".